docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -564,18 +567,69 @@ private synchronized void load() {
             LOGGER.log(Level.SEVERE, "Failed to load the configuration from config path = "+config,e);
         }
     }
-    
+
+    /**
+     * Represents the different response formats (JSON, pretty JSON, XML).
+     */
+    private enum ResponseFormat {
```

Comment:
why not just use the enum value in the web request and get stapler to parse and covert it for you?